src/patterns/check-a-service-is-suitable/index.md.njk

```diff
@@ -14,7 +14,7 @@ Doing this helps users save time as it tells them upfront whether they are eligi
 
 It can also help reduce time and money spent processing queries from users confused about whether they’re eligible to use your service, or if it’s suitable for them.
 
-![‘Check a service is suitable’ flow diagram. Contains an introduction page followed by a series of simple questions. If at any point a user is deemed not eligible for the service they will be pointed to a page that explains why they are not eligible. Otherwise they will be presented a ‘results’ page. ](check-a-service-is-suitable-flow.png)
+![‘Check a service is suitable’ flow diagram. Contains an introduction page followed by a series of simple questions. If at any point a user is deemed not eligible for the service they will be pointed to a page that explains why they are not eligible. Otherwise they will be presented a ‘results’ page. ](check-a-service-is-suitable-flow.svg)
```

Comment:
The filename of the new SVG is `check-a-service-suitable-flow.svg` – without the `-is-` – so either this needs updating or the file needs renaming.

(This is why the tests are failing at the minute)